lib/sitemap/format_boost_calculator.rb

```diff
@@ -0,0 +1,20 @@
+class FormatBoostCalculator
+  FORMAT_BOOST_CONFIG = YAML.load_file('config/query/format_boosting.yml')
+
+  def initialize
+    @format_boosts = FORMAT_BOOST_CONFIG["format_boosts"]
+  end
+
+  def boost(format)
+    page_boost = @format_boosts[format] || 1
+
+    # Normalise format boost to always give a value between 0 and 1
+    page_boost / max_boost
```

Comment:
👍 I've updated it and added a test.

I've also added the rounding that we discussed in person: all the priorities are now rounded to two decimal places.